src/v/pandaproxy/schema_registry/test/store.cc

```diff
@@ -63,98 +63,6 @@ BOOST_AUTO_TEST_CASE(test_store_insert) {
     BOOST_REQUIRE_EQUAL(ins_res.version, pps::schema_version{2});
 }
 
-BOOST_AUTO_TEST_CASE(test_store_upsert_in_order) {
```

Comment:
Are these no longer useful?

Reply:
These were testing a method that's never used outside of tests -- do you think it's worth keeping that method just to provide a test hook to exercise the underlying upsert_schema/upsert_subject functions?

Reply:
Reinstated these tests, moved the store upsert method into the test as a helper.